app/ui/src/app/integration/edit-page/step-configure/templater/templater.component.html

```diff
@@ -3,7 +3,8 @@
 
     <div class="form-group">
 
-      <legend>Choose how you want to add a template:</legend>
+      <label></label>
+      <label>{{ 'integrations.steps.templater-choice' | synI18n }}</label>
```

Comment:
You can just use `<strong>` here instead of `<label>`.

Reply:
I chatted with @dongniwang about it, and we can use `<strong>` there (`bold`). We use that in other places in the app, too (`/customizations/api-connector/create/swagger-connector` and `/support` for example).